R/MsBackend.R

```diff
@@ -881,6 +874,13 @@ setMethod("precursorMz", "MsBackend", function(object) {
     stop("Not implemented for ", class(object), ".")
 })
 
+#' @exportMethod replaceList
+#'
+#' @rdname MsBackend
+setMethod("replaceList", "MsBackend", function(object, value, ...) {
```

Comment:
Could/Shouldn't this be a `setReplaceMethod` as well?

Reply:
I followed what they did in `S4Vectors`. The `replaceROW` and `replaceCOL` are **not** replace methods, they're just simple methods. I have no strong opinion here. If you want me to change I'll do.

Reply:
Me neither. Is there a performance drop? Or is `setReplaceMethod` internally just a synonym for `object <- setMethod`? (I don't have access to R at the moment.)

Reply:
Could one advantage be to have a dispatch on both `object` and `value`?

Reply:
Anyway, I'm going to change it to a replace method. Usage will then be:

```r
lst <- as.list(be)
replaceList(be) <- lst
```

I think that might be easier for users/developers to understand.